File: Holiday_API/Holiday_API/Holiday.cs
﻿using System;
using System.ComponentModel.DataAnnotations;

namespace Holiday_API
{
    public class Holiday
    {
        public int ID { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Date { get; set; }
    }
}

File: Holiday_App_Android/app/src/main/java/com/example/holiday_app_android/models/Holiday.java
package com.example.holiday_app_android.models;

import org.json.JSONException;
import org.json.JSONObject;

public class Holiday {
    private String Name;
    private String Date;

    @SuppressWarnings("unused")
    public Holiday(JSONObject object)
    {
        try {
            String ID = object.getString("id");
            this.Name = object.getString("name");
            this.Date = object.getString("date");

        } catch (JSONException e) {
            e.printStackTrace();
        }
    }

    public Holiday(String Name, String Date)
    {
        this.Name = Name;
        this.Date = Date;
    }

    public String getName() {
        return Name;
    }

    public String getDate() {
        return Date;
    }

}
